core/node/da_clients/src/eigen/sdk.rs

```diff
@@ -1,76 +1,150 @@
-use std::{str::FromStr, time::Duration};
+use std::{str::FromStr, sync::Arc};
 
+use anyhow::Context;
 use secp256k1::{ecdsa::RecoverableSignature, SecretKey};
 use tokio::sync::mpsc;
-use tokio_stream::{wrappers::ReceiverStream, StreamExt};
+use tokio_stream::{wrappers::UnboundedReceiverStream, StreamExt};
 use tonic::{
     transport::{Channel, ClientTlsConfig, Endpoint},
     Streaming,
 };
-
+use url::Url;
+use zksync_config::EigenConfig;
+use zksync_da_client::types::DAError;
+use zksync_eth_client::clients::PKSigningClient;
+use zksync_types::{url::SensitiveUrl, K256PrivateKey, SLChainId};
+use zksync_web3_decl::client::{Client, DynClient, L1};
+
+use super::{
+    blob_info::BlobInfo,
+    disperser::BlobInfo as DisperserBlobInfo,
+    verifier::{Verifier, VerifierConfig},
+    GetBlobData,
+};
 use crate::eigen::{
-    disperser,
+    blob_info,
     disperser::{
+        self,
         authenticated_request::Payload::{AuthenticationData, DisperseRequest},
         disperser_client::DisperserClient,
-        AuthenticatedReply, BlobAuthHeader, BlobVerificationProof, DisperseBlobReply,
+        AuthenticatedReply, BlobAuthHeader,
     },
+    verifier::VerificationError,
 };
 
 #[derive(Debug, Clone)]
```

Comment:
Dumb question: Is client required to be cloneable by some external code? In the workspace, it seems to work fine with `derive(Clone)` removed (ditto for `Verifier`).

Reply:
Fixed here https://github.com/lambdaclass/zksync-era/pull/386